lib/v2/notion/database.js

```diff
@@ -0,0 +1,160 @@
+const { Client, isNotionClientError, APIErrorCode } = require('@notionhq/client');
+const logger = require('@/utils/logger');
+const config = require('@/config').value;
+const { parseDate } = require('@/utils/parse-date');
+const got = require('@/utils/got');
+const { NotionToMarkdown } = require("notion-to-md");
+const cheerio = require('cheerio');
+const md = require('markdown-it')({
+    html: true,
+    linkify: true,
+});
+
+module.exports = async (ctx) => {
```

Comment:
https://github.com/DIYgod/RSSHub/blob/9bdba060c7962d734745c8d6b9264da9a532b51d/lib/routes/disqus/posts.js#L5-L7
Similarly, you can throw an error if the token is not set

Reply:
Done, I've added notion key check